src/box/lua/config/applier/roles.lua

```diff
@@ -0,0 +1,62 @@
+local log = require('internal.config.utils.log')
+
+local last_loaded = {}
+
+local function stop_roles(roles)
```

Comment:
It is not obvious that the `roles` arguments means roles to skip, not to stop.

Reply:
Fixed.